scripts/test/Muon/home_runinfo_presenter_test.py

```diff
@@ -46,14 +42,22 @@ def test_runinfo_correct(self):
 
         self.presenter.update_view_from_model()
 
-        expected_string_list = ['Instrument:MUSR', 'Run:22725', 'Title:FeTeSeT=1F=100', 'Comment:FCfirstsample',
-                                'Start:2009-03-24T04:18:58', 'End:2009-03-24T04:56:26', 'Counts(MEv):20.076704',
-                                'GoodFrames:88540', 'CountsperGoodFrame:226.753',
-                                'CountsperGoodFrameperdet:3.543', 'AverageTemperature(K):19.69992',
-                                'SampleTemperature(K):1.0', 'SampleMagneticField(G):100.0',
-                                'NumberofDAQPeriods:1']
+        add_text_line_calls = [mock.call("Instrument                : MUSR"),
+                               mock.call("Run                       : 22725"),
+                               mock.call("Title                     : FeTeSe T=1 F=100"),
+                               mock.call("Comment                   : FC first sample"),
+                               mock.call("Start                     : 2009-03-24T04:18:58"),
+                               mock.call("End                       : 2009-03-24T04:56:26"),
+                               mock.call("Counts (MEv)              : 20.076704"),
+                               mock.call("Good Frames               : 88540"),
+                               mock.call("Counts per Good Frame     : 226.753"),
+                               mock.call("Counts per Good Frame per det : 3.543"),
+                               mock.call("Average Temperature (K)   : 19.69992"),
+                               mock.call("Sample Temperature (K)    : 1.0"),
+                               mock.call("Sample Magnetic Field (G) : 100.0"),
+                               mock.call("Number of DAQ Periods     : 1")]
```

Comment:
I think you can avoid making all these `call` objects by comparing against `mock.call_args_list` with a normal list https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args_list

Reply:
It appears you still need to wrap them in the `call` object for the assertion to work. The documentation for this seems a little misleading

Reply:
Ah fair enough